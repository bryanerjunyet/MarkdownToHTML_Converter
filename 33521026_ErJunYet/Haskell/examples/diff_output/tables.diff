diff --git a/examples/expected_output/tables.html b/examples/output/tables.html
index 70fde47..57c7ced 100644
--- a/examples/expected_output/tables.html
+++ b/examples/output/tables.html
@@ -9,28 +9,9 @@
 <body>
     <table>
         <thead>
-            <tr>
-                <th>Tables</th>
-                <th>Are</th>
-                <th>Cool</th>
-            </tr>
+    <tr>    <td> Tables        </td>    <td> Are           </td>    <td> Cool  </td></tr>
         </thead>
         <tbody>
-            <tr>
-                <td>here</td>
-                <td>is</td>
-                <td>data</td>
-            </tr>
-            <tr>
-                <td>here</td>
-                <td>is</td>
-                <td>data</td>
-            </tr>
-            <tr>
-                <td>here</td>
-                <td>is also</td>
-                <td><strong>bolded data</strong></td>
-            </tr>
         </tbody>
     </table>
 </body>
