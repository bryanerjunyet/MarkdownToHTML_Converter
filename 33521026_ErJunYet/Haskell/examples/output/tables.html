<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Test</title>
</head>

<body>
    <table>
        <thead>
    <tr>    <td> Tables        </td>    <td> Are           </td>    <td> Cool  </td></tr>
        </thead>
        <tbody>
        </tbody>
    </table>
</body>

</html>
