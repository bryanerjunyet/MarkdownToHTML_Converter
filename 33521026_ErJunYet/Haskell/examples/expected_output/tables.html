<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Test</title>
</head>

<body>
    <table>
        <thead>
            <tr>
                <th>Tables</th>
                <th>Are</th>
                <th>Cool</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>here</td>
                <td>is</td>
                <td>data</td>
            </tr>
            <tr>
                <td>here</td>
                <td>is</td>
                <td>data</td>
            </tr>
            <tr>
                <td>here</td>
                <td>is also</td>
                <td><strong>bolded data</strong></td>
            </tr>
        </tbody>
    </table>
</body>

</html>
